resources/metadata/codeqlExecuteScan.yaml

```diff
@@ -105,6 +105,20 @@ spec:
           - STAGES
           - STEPS
         default: false
+      - name: threads
```

Comment:
Can you please set the default value here as 0? 
Remove "Default - 0" from the description.